inngest/src/main/kotlin/com/inngest/InngestFunctionConfigBuilder.kt

```diff
@@ -119,18 +120,21 @@ class InngestFunctionConfigBuilder {
         return this
     }
 
+    /**
+     * Specifies the maximum number of retries for all steps across this function.
+     *
+     * @param attempts The number of times to retry a step before failing, defaults to 3.
+     */
+    fun retries(attempts: Int): InngestFunctionConfigBuilder = apply { this.retries = attempts }
```

Comment:
cool TIL about [apply](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/apply.html), maybe worth applying it elsewhere in the builder later